src/applications/dhp-connected-devices/components/DeviceConnectionAlerts.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export const DeviceConnectionSucceededAlert = () => {
+  return (
+    <div data-testId="success-alert">
+      <va-alert
+        close-btn-aria-label="Close notification"
+        status="success"
+        visible
+      >
+        <h3 slot="headline">Device Connected</h3>
+        <div>Your device is now connected.</div>
+      </va-alert>
+    </div>
+  );
+};
+
+export const DeviceConnectionFailedAlert = () => {
+  return (
+    <div data-testId="failure-alert">
+      <va-alert
+        close-btn-aria-label="Close notification"
+        status="error"
+        visible
+      >
+        <h3 slot="headline">We couldn’t connect your device</h3>
+        <div>
+          We were not able to connect to your device now. Please try again.
```

Comment:
Same thing here. The wire frames show `We were not able to connect to your device _right_ now`. Totally looks good to me, just an FYI.

Reply:
Good catch, let's update to "We were not able to connect to your device right now. Please try again."